### ISSUE TITLE
Existing videos are overwritten even with --no-overwrites

### ISSUE THREAD

**Issue by KrischnaGabriel** [yt-dlp/yt-dlp#6416]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting a bug unrelated to a specific site
- [X] I've verified that I'm running yt-dlp version **2023.03.03** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)

### Provide a description that is worded well enough to be understood

When downloading videos they are re-downloaded and overwritten always.
This is especially annoying when downloading a playlist and the yt-dlp process terminates because of an error (example: no network connection) or when the yt-dlp process is terminated for other reasons. To continue downloading it is then either required to re-download the entire playlist or manually exclude the already downloaded videos.
The `--no-force-overwrites` argument which is enabled by default should prevent this, but it's not. Also `-w` aka `--no-overwrites` does not prevent the overwriting of existing video/music files (See verbose output).

About the verbose output:
Before, the same command was run to download the first three videos. The verbose output provided thus was run with the first three videos pre-existing. 

PS: I usually run the pip version of yt-dlp, but pip does not has the most recent version yet.

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [X] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
~/Music/gta$ ./yt-dlp_linux -x --embed-thumbnail -vU -w https://m.youtube.com/watch?list=PLADajjvOG11G3_tD3_Teka1cAky1r5rZQ
[debug] Command-line config: ['-x', '--embed-thumbnail', '-vU', '-w', 'https://m.youtube.com/watch?list=PLADajjvOG11G3_tD3_Teka1cAky1r5rZQ']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version 2023.03.03 [934496428] (linux_exe)
[debug] Python 3.10.9 (CPython x86_64 64bit) - Linux-5.4.0-136-generic-x86_64-with-glibc2.31 (OpenSSL 3.0.8 7 Feb 2023, glibc 2.31)
[debug] exe versions: ffmpeg 4.2.7, ffprobe 4.2.7
[debug] Optional libraries: Cryptodome-3.16.0, brotli-1.0.9, certifi-2022.12.07, mutagen-1.46.0, sqlite3-2.6.0, websockets-10.4
[debug] Proxy map: {}
[debug] Loaded 1782 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Available version: stable@2023.03.03, Current version: stable@2023.03.03
Current Build Hash: 98aafceec14aada2b547a6f2e927066db6af86db8e61d0fd0aa113a235f3479c
yt-dlp is up to date (stable@2023.03.03)
[youtube:tab] Extracting URL: https://m.youtube.com/watch?list=PLADajjvOG11G3_tD3_Teka1cAky1r5rZQ
WARNING: [youtube:tab] A video URL was given without video ID. Trying to download playlist PLADajjvOG11G3_tD3_Teka1cAky1r5rZQ
[youtube:tab] Extracting URL: https://www.youtube.com/playlist?list=PLADajjvOG11G3_tD3_Teka1cAky1r5rZQ
[youtube:tab] PLADajjvOG11G3_tD3_Teka1cAky1r5rZQ: Downloading webpage
[youtube:tab] PLADajjvOG11G3_tD3_Teka1cAky1r5rZQ: Redownloading playlist API JSON with unavailable videos
[download] Downloading playlist: Grand Theft Auto [GTA] V Soundtracks
[youtube:tab] PLADajjvOG11G3_tD3_Teka1cAky1r5rZQ page 1: Downloading API JSON
[youtube:tab] PLADajjvOG11G3_tD3_Teka1cAky1r5rZQ page 2: Downloading API JSON
[debug] Skipping writing playlist thumbnail
[youtube:tab] Playlist Grand Theft Auto [GTA] V Soundtracks: Downloading 230 items of 230
[download] Downloading item 1 of 230
[youtube] Extracting URL: https://www.youtube.com/watch?v=KzKvPrIPVbE
[youtube] KzKvPrIPVbE: Downloading webpage
[youtube] KzKvPrIPVbE: Downloading android player API JSON
[debug] Sort order given by extractor: quality, res, fps, hdr:12, source, vcodec:vp9.2, channels, acodec, lang, proto
[debug] Formats sorted by: hasvid, ie_pref, quality, res, fps, hdr:12(7), source, vcodec:vp9.2(10), channels, acodec, lang, proto, filesize, fs_approx, tbr, vbr, abr, asr, vext, aext, hasaud, id
[info] KzKvPrIPVbE: Downloading 1 format(s): 251
[info] Downloading video thumbnail 46 ...
[info] Writing video thumbnail 46 to: GTA V - Welcome to Los Santos Soundtrack - Intro⧸Theme song [KzKvPrIPVbE].webp
[debug] Invoking dashsegments downloader on "https://rr2---sn-8xgn5uxa-4g56.googlevideo.com/videoplayback?expire=1677954966&ei=NjsDZMKDOYSa1gL246OQBA&ip=94.134.114.37&id=o-ANZHSaXGcHWGHqdDyKupKXruMw8L7y5S0pkHoqfbIDPR&itag=251&source=youtube&requiressl=yes&mh=DX&mm=31%2C29&mn=sn-8xgn5uxa-4g56%2Csn-4g5edn6r&ms=au%2Crdu&mv=m&mvi=2&pcm2cms=yes&pl=24&initcwndbps=2005000&spc=H3gIhlg6QruxL_M9Pm9O12-JHwTyoiM&vprv=1&svpuc=1&mime=audio%2Fwebm&gir=yes&clen=2452927&dur=149.441&lmt=1576468880644132&mt=1677932940&fvip=2&keepalive=yes&fexp=24007246&c=ANDROID&txp=5531432&sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cspc%2Cvprv%2Csvpuc%2Cmime%2Cgir%2Cclen%2Cdur%2Clmt&sig=AOq0QJ8wRgIhANRz44U8-UzYa-BfQQutyzWv6NjzTSO87yfcXVgCG52iAiEAt5TcKE8ziK8hTOMK4BwkA_pP4g58youCnzCYRXpoTQo%3D&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpcm2cms%2Cpl%2Cinitcwndbps&lsig=AG3C_xAwRgIhAIe9vyOWM4L_TEPOujYLwgj0-21hSM-RZWkaEtq80WQOAiEAz75ueN6Q-YVWl1Xa7UvgMYbuCaPvkyhZuo8iwsYQnTs%3D"
[dashsegments] Total fragments: 1
[download] Destination: GTA V - Welcome to Los Santos Soundtrack - Intro⧸Theme song [KzKvPrIPVbE].webm
[download] 100% of    2.34MiB in 00:00:00 at 3.52MiB/s
[debug] ffmpeg command line: ffprobe -show_streams 'file:GTA V - Welcome to Los Santos Soundtrack - Intro⧸Theme song [KzKvPrIPVbE].webm'
[ExtractAudio] Destination: GTA V - Welcome to Los Santos Soundtrack - Intro⧸Theme song [KzKvPrIPVbE].opus
[debug] ffmpeg command line: ffmpeg -y -loglevel repeat+info -i 'file:GTA V - Welcome to Los Santos Soundtrack - Intro⧸Theme song [KzKvPrIPVbE].webm' -vn -acodec copy -movflags +faststart 'file:GTA V - Welcome to Los Santos Soundtrack - Intro⧸Theme song [KzKvPrIPVbE].opus'
Deleting original file GTA V - Welcome to Los Santos Soundtrack - Intro⧸Theme song [KzKvPrIPVbE].webm (pass -k to keep)
[ThumbnailsConvertor] Converting thumbnail "GTA V - Welcome to Los Santos Soundtrack - Intro⧸Theme song [KzKvPrIPVbE].webp" to png
[debug] ffmpeg command line: ffmpeg -y -loglevel repeat+info -f image2 -pattern_type none -i 'file:GTA V - Welcome to Los Santos Soundtrack - Intro⧸Theme song [KzKvPrIPVbE].webp' -update 1 -movflags +faststart 'file:GTA V - Welcome to Los Santos Soundtrack - Intro⧸Theme song [KzKvPrIPVbE].png'
[EmbedThumbnail] mutagen: Adding thumbnail to "GTA V - Welcome to Los Santos Soundtrack - Intro⧸Theme song [KzKvPrIPVbE].opus"
[debug] ffmpeg command line: ffmpeg -y -loglevel repeat+info -i 'file:GTA V - Welcome to Los Santos Soundtrack - Intro⧸Theme song [KzKvPrIPVbE].png'
[download] Downloading item 2 of 230
[youtube] Extracting URL: https://www.youtube.com/watch?v=2MUFNOizFUw
[youtube] 2MUFNOizFUw: Downloading webpage
[youtube] 2MUFNOizFUw: Downloading android player API JSON
[debug] Sort order given by extractor: quality, res, fps, hdr:12, source, vcodec:vp9.2, channels, acodec, lang, proto
[debug] Formats sorted by: hasvid, ie_pref, quality, res, fps, hdr:12(7), source, vcodec:vp9.2(10), channels, acodec, lang, proto, filesize, fs_approx, tbr, vbr, abr, asr, vext, aext, hasaud, id
[info] 2MUFNOizFUw: Downloading 1 format(s): 251
[info] Downloading video thumbnail 46 ...
[info] Writing video thumbnail 46 to: GTA V ｜ Loading Screen Song [2MUFNOizFUw].webp
[debug] Invoking dashsegments downloader on "https://rr4---sn-8xgn5uxa-4g5l.googlevideo.com/videoplayback?expire=1677954969&ei=OTsDZPSmC4K18gPj9peQCQ&ip=94.134.114.37&id=o-AJ-TZHBUlTooEjKZimZkxzqep9QUiPsDjQyb8xrAcmY3&itag=251&source=youtube&requiressl=yes&mh=tF&mm=31%2C29&mn=sn-8xgn5uxa-4g5l%2Csn-4g5lznl6&ms=au%2Crdu&mv=m&mvi=4&pl=24&initcwndbps=1972500&spc=H3gIhnC_u1NjXUtxM0mBmoTW9ND7IuA&vprv=1&svpuc=1&mime=audio%2Fwebm&gir=yes&clen=7634678&dur=426.061&lmt=1553733060252175&mt=1677932940&fvip=2&keepalive=yes&fexp=24007246&c=ANDROID&txp=1311222&sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cspc%2Cvprv%2Csvpuc%2Cmime%2Cgir%2Cclen%2Cdur%2Clmt&sig=AOq0QJ8wRgIhAK8bsF8NKuH9vkehe1pcqZ9aGQiO2Pg0oQiVs6E7yE81AiEAsmjaRWrYtb5aLUpxGwy5QtHKRxXcRCPdNaGrt4oIoQ8%3D&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps&lsig=AG3C_xAwRAIgdGN5u8AG_O09p61liVha1eyTtiAjaWRYNuDN2stptDMCIESyuZ-jEtK_5U6CU4SEhIL11oyB_KcH7Se9qGxgcheW"
[dashsegments] Total fragments: 1
[download] Destination: GTA V ｜ Loading Screen Song [2MUFNOizFUw].webm
[download] 100% of    7.28MiB in 00:00:01 at 4.08MiB/s
[debug] ffmpeg command line: ffprobe -show_streams 'file:GTA V ｜ Loading Screen Song [2MUFNOizFUw].webm'
[ExtractAudio] Destination: GTA V ｜ Loading Screen Song [2MUFNOizFUw].opus
[debug] ffmpeg command line: ffmpeg -y -loglevel repeat+info -i 'file:GTA V ｜ Loading Screen Song [2MUFNOizFUw].webm' -vn -acodec copy -movflags +faststart 'file:GTA V ｜ Loading Screen Song [2MUFNOizFUw].opus'
Deleting original file GTA V ｜ Loading Screen Song [2MUFNOizFUw].webm (pass -k to keep)
[ThumbnailsConvertor] Converting thumbnail "GTA V ｜ Loading Screen Song [2MUFNOizFUw].webp" to png
[debug] ffmpeg command line: ffmpeg -y -loglevel repeat+info -f image2 -pattern_type none -i 'file:GTA V ｜ Loading Screen Song [2MUFNOizFUw].webp' -update 1 -movflags +faststart 'file:GTA V ｜ Loading Screen Song [2MUFNOizFUw].png'
[EmbedThumbnail] mutagen: Adding thumbnail to "GTA V ｜ Loading Screen Song [2MUFNOizFUw].opus"
[debug] ffmpeg command line: ffmpeg -y -loglevel repeat+info -i 'file:GTA V ｜ Loading Screen Song [2MUFNOizFUw].png'
[download] Downloading item 3 of 230
[youtube] Extracting URL: https://www.youtube.com/watch?v=jIGpIBiaZb0
[youtube] jIGpIBiaZb0: Downloading webpage
[youtube] jIGpIBiaZb0: Downloading android player API JSON
[debug] Sort order given by extractor: quality, res, fps, hdr:12, source, vcodec:vp9.2, channels, acodec, lang, proto
[debug] Formats sorted by: hasvid, ie_pref, quality, res, fps, hdr:12(7), source, vcodec:vp9.2(10), channels, acodec, lang, proto, filesize, fs_approx, tbr, vbr, abr, asr, vext, aext, hasaud, id
[info] jIGpIBiaZb0: Downloading 1 format(s): 251
[info] Downloading video thumbnail 46 ...
^C
ERROR: Interrupted by user
```

**Comment by pukkandan** [comment]
The thumbnail files don't exist and so are not being overwritten. What you want is https://github.com/yt-dlp/yt-dlp/issues/3206

**Comment by KrischnaGabriel** [comment]
> The thumbnail files don't exist and so are not being overwritten. What you are asking for is #3206

Even without the `--embed-thumbnail` argument existing videos are overwritten.
This is an unrelated issue

**Comment by pukkandan** [comment]
log?

**Comment by pukkandan** [comment]
hm.. I misunderstood the log. yt-dlp doesnt think the files exist. Can show `ls && yt-dlp -vU "URL"`

**Comment by KrischnaGabriel** [comment]
yes, here:
```
$ ls && ./yt-dlp_linux -vU -x https://m.youtube.com/watch?list=PLADajjvOG11G3_tD3_Teka1cAky1r5rZQ
'Grand Theft Auto [GTA] V - Prologue (1st Mission) Music Theme [jIGpIBiaZb0].opus'
'GTA V ｜ Loading Screen Song [2MUFNOizFUw].opus'
'GTA V - Welcome to Los Santos Soundtrack - Intro⧸Theme song [KzKvPrIPVbE].opus'
 yt-dlp_linux
[debug] Command-line config: ['-vU', '-x', 'https://m.youtube.com/watch?list=PLADajjvOG11G3_tD3_Teka1cAky1r5rZQ']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version 2023.03.03 [934496428] (linux_exe)
[debug] Python 3.10.9 (CPython x86_64 64bit) - Linux-5.4.0-136-generic-x86_64-with-glibc2.31 (OpenSSL 3.0.8 7 Feb 2023, glibc 2.31)
[debug] exe versions: ffmpeg 4.2.7, ffprobe 4.2.7
[debug] Optional libraries: Cryptodome-3.16.0, brotli-1.0.9, certifi-2022.12.07, mutagen-1.46.0, sqlite3-2.6.0, websockets-10.4
[debug] Proxy map: {}
[debug] Loaded 1782 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Available version: stable@2023.03.03, Current version: stable@2023.03.03
Current Build Hash: 98aafceec14aada2b547a6f2e927066db6af86db8e61d0fd0aa113a235f3479c
yt-dlp is up to date (stable@2023.03.03)
[youtube:tab] Extracting URL: https://m.youtube.com/watch?list=PLADajjvOG11G3_tD3_Teka1cAky1r5rZQ
WARNING: [youtube:tab] A video URL was given without video ID. Trying to download playlist PLADajjvOG11G3_tD3_Teka1cAky1r5rZQ
[youtube:tab] Extracting URL: https://www.youtube.com/playlist?list=PLADajjvOG11G3_tD3_Teka1cAky1r5rZQ
[youtube:tab] PLADajjvOG11G3_tD3_Teka1cAky1r5rZQ: Downloading webpage
[youtube:tab] PLADajjvOG11G3_tD3_Teka1cAky1r5rZQ: Redownloading playlist API JSON with unavailable videos
[download] Downloading playlist: Grand Theft Auto [GTA] V Soundtracks
[youtube:tab] PLADajjvOG11G3_tD3_Teka1cAky1r5rZQ page 1: Downloading API JSON
[youtube:tab] PLADajjvOG11G3_tD3_Teka1cAky1r5rZQ page 2: Downloading API JSON
[youtube:tab] Playlist Grand Theft Auto [GTA] V Soundtracks: Downloading 230 items of 230
[download] Downloading item 1 of 230
[youtube] Extracting URL: https://www.youtube.com/watch?v=KzKvPrIPVbE
[youtube] KzKvPrIPVbE: Downloading webpage
[youtube] KzKvPrIPVbE: Downloading android player API JSON
[debug] Sort order given by extractor: quality, res, fps, hdr:12, source, vcodec:vp9.2, channels, acodec, lang, proto
[debug] Formats sorted by: hasvid, ie_pref, quality, res, fps, hdr:12(7), source, vcodec:vp9.2(10), channels, acodec, lang, proto, filesize, fs_approx, tbr, vbr, abr, asr, vext, aext, hasaud, id
[info] KzKvPrIPVbE: Downloading 1 format(s): 251
[debug] Invoking dashsegments downloader on "https://rr2---sn-8xgn5uxa-4g56.googlevideo.com/videoplayback?expire=1677957135&ei=r0MDZN2xHPTYx_APj92lqAw&ip=94.134.114.37&id=o-AGEQJ5aKARBaQ1robXcYErST6MRZwk-OvdpfGEiPsiI7&itag=251&source=youtube&requiressl=yes&mh=DX&mm=31%2C29&mn=sn-8xgn5uxa-4g56%2Csn-4g5e6nzl&ms=au%2Crdu&mv=m&mvi=2&pl=24&initcwndbps=2142500&spc=H3gIhlEHeqGevcT-CAC7sjou24bTes0&vprv=1&svpuc=1&mime=audio%2Fwebm&gir=yes&clen=2452927&dur=149.441&lmt=1576468880644132&mt=1677935112&fvip=3&keepalive=yes&fexp=24007246&c=ANDROID&txp=5531432&sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cspc%2Cvprv%2Csvpuc%2Cmime%2Cgir%2Cclen%2Cdur%2Clmt&sig=AOq0QJ8wRAIgbTRSXNPyqbzGP1CDFKmh4sURJuv19pB1YQr7AfBTTQACICqlt2sXlPqoLyKUrhe5TGXlulvRXjxrQHZpsm2LWJ64&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps&lsig=AG3C_xAwRAIgGxXtrHU1DVPzoJR0in_NGRT5QrRdfom_TLc0ZNDGgj4CIBLhJKqWfoGcgvi3wcd_4bgCEJ7oAR2eobcWnklQAHHP"
[dashsegments] Total fragments: 1
[download] Destination: GTA V - Welcome to Los Santos Soundtrack - Intro⧸Theme song [KzKvPrIPVbE].webm
[download] 100% of    2.34MiB in 00:00:00 at 3.74MiB/s
[debug] ffmpeg command line: ffprobe -show_streams 'file:GTA V - Welcome to Los Santos Soundtrack - Intro⧸Theme song [KzKvPrIPVbE].webm'
[ExtractAudio] Destination: GTA V - Welcome to Los Santos Soundtrack - Intro⧸Theme song [KzKvPrIPVbE].opus
[debug] ffmpeg command line: ffmpeg -y -loglevel repeat+info -i 'file:GTA V - Welcome to Los Santos Soundtrack - Intro⧸Theme song [KzKvPrIPVbE].webm' -vn -acodec copy -movflags +faststart 'file:GTA V - Welcome to Los Santos Soundtrack - Intro⧸Theme song [KzKvPrIPVbE].opus'
Deleting original file GTA V - Welcome to Los Santos Soundtrack - Intro⧸Theme song [KzKvPrIPVbE].webm (pass -k to keep)
[download] Downloading item 2 of 230
[youtube] Extracting URL: https://www.youtube.com/watch?v=2MUFNOizFUw
[youtube] 2MUFNOizFUw: Downloading webpage
[youtube] 2MUFNOizFUw: Downloading android player API JSON
[debug] Sort order given by extractor: quality, res, fps, hdr:12, source, vcodec:vp9.2, channels, acodec, lang, proto
[debug] Formats sorted by: hasvid, ie_pref, quality, res, fps, hdr:12(7), source, vcodec:vp9.2(10), channels, acodec, lang, proto, filesize, fs_approx, tbr, vbr, abr, asr, vext, aext, hasaud, id
[info] 2MUFNOizFUw: Downloading 1 format(s): 251
[debug] Invoking dashsegments downloader on "https://rr4---sn-8xgn5uxa-4g5l.googlevideo.com/videoplayback?expire=1677957137&ei=sUMDZJS9JY-s7gPWzLKgDA&ip=94.134.114.37&id=o-AAkPjiBLkn7aphAlpurdfjFTfQ6DXYNfHFr5egftFv49&itag=251&source=youtube&requiressl=yes&mh=tF&mm=31%2C29&mn=sn-8xgn5uxa-4g5l%2Csn-4g5lznl6&ms=au%2Crdu&mv=m&mvi=4&pl=24&initcwndbps=2001250&spc=H3gIhuzsq7ArUkR0QOTksOq6zQxoaGI&vprv=1&svpuc=1&mime=audio%2Fwebm&gir=yes&clen=7634678&dur=426.061&lmt=1553733060252175&mt=1677935112&fvip=2&keepalive=yes&fexp=24007246&c=ANDROID&txp=1311222&sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cspc%2Cvprv%2Csvpuc%2Cmime%2Cgir%2Cclen%2Cdur%2Clmt&sig=AOq0QJ8wRAIgfYA80UHeb4L7Y2rAY2Cdr5iQ1sr0X0UsUjYzKGPXUFoCIGgsbvI2f19h4aFSEC1qzDKlJ5RexzMw8Js_UwdCHj65&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps&lsig=AG3C_xAwRQIhAIpu2uYVa_Ja-XjhpgS-mYWQ-WuzQh4ObG1xW7ZGXJYhAiBnwjXaY2VP8_VL5zPaIny_PhOPWNqxCngMlRLAg4_OAA%3D%3D"
[dashsegments] Total fragments: 1
[download] Destination: GTA V ｜ Loading Screen Song [2MUFNOizFUw].webm
[download] 100% of    7.28MiB in 00:00:01 at 3.99MiB/s
[debug] ffmpeg command line: ffprobe -show_streams 'file:GTA V ｜ Loading Screen Song [2MUFNOizFUw].webm'
[ExtractAudio] Destination: GTA V ｜ Loading Screen Song [2MUFNOizFUw].opus
[debug] ffmpeg command line: ffmpeg -y -loglevel repeat+info -i 'file:GTA V ｜ Loading Screen Song [2MUFNOizFUw].webm' -vn -acodec copy -movflags +faststart 'file:GTA V ｜ Loading Screen Song [2MUFNOizFUw].opus'
Deleting original file GTA V ｜ Loading Screen Song [2MUFNOizFUw].webm (pass -k to keep)
[download] Downloading item 3 of 230
[youtube] Extracting URL: https://www.youtube.com/watch?v=jIGpIBiaZb0
[youtube] jIGpIBiaZb0: Downloading webpage
[youtube] jIGpIBiaZb0: Downloading android player API JSON
[debug] Sort order given by extractor: quality, res, fps, hdr:12, source, vcodec:vp9.2, channels, acodec, lang, proto
[debug] Formats sorted by: hasvid, ie_pref, quality, res, fps, hdr:12(7), source, vcodec:vp9.2(10), channels, acodec, lang, proto, filesize, fs_approx, tbr, vbr, abr, asr, vext, aext, hasaud, id
[info] jIGpIBiaZb0: Downloading 1 format(s): 251
[debug] Invoking dashsegments downloader on "https://rr3---sn-8xgn5uxa-4g56.googlevideo.com/videoplayback?expire=1677957140&ei=tEMDZL6QI57Fx_APwOyqgAM&ip=94.134.114.37&id=o-ANWC3SQsgpmQ_AaxOYjAbQuiux8PdnVPi7j2oXD5W8BJ&itag=251&source=youtube&requiressl=yes&mh=d5&mm=31%2C29&mn=sn-8xgn5uxa-4g56%2Csn-4g5lzned&ms=au%2Crdu&mv=m&mvi=3&pl=24&initcwndbps=2142500&spc=H3gIhvMK2jC-lcOx1qYig_tBrz4tJwY&vprv=1&svpuc=1&mime=audio%2Fwebm&gir=yes&clen=31440681&dur=1723.721&lmt=1496238345395087&mt=1677935112&fvip=3&keepalive=yes&fexp=24007246&c=ANDROID&sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cspc%2Cvprv%2Csvpuc%2Cmime%2Cgir%2Cclen%2Cdur%2Clmt&sig=AOq0QJ8wRgIhAOWEvdCJ0pctVWgyu8gdi-wehdYeAIeIFumlmuDgvLOcAiEAhAKgI1TrLbm1hQaPQN4k-IRdDQZ46sR42oYelpBwb2Y%3D&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps&lsig=AG3C_xAwRQIhAKULG5hPImzRhI5hlETCyXhaQgWRuAvGNzkziyHkb4zMAiBPvIFrCF8IQBFcsdkkJ92wufIDj52lfSBKdsY9956iBA%3D%3D"
[dashsegments] Total fragments: 3
[download] Destination: Grand Theft Auto [GTA] V - Prologue (1st Mission) Music Theme [jIGpIBiaZb0].webm
[download] 100% of   29.98MiB in 00:00:08 at 3.50MiB/s
[debug] ffmpeg command line: ffprobe -show_streams 'file:Grand Theft Auto [GTA] V - Prologue (1st Mission) Music Theme [jIGpIBiaZb0].webm'
[ExtractAudio] Destination: Grand Theft Auto [GTA] V - Prologue (1st Mission) Music Theme [jIGpIBiaZb0].opus
[debug] ffmpeg command line: ffmpeg -y -loglevel repeat+info -i 'file:Grand Theft Auto [GTA] V - Prologue (1st Mission) Music Theme [jIGpIBiaZb0].webm' -vn -acodec copy -movflags +faststart 'file:Grand Theft Auto [GTA] V - Prologue (1st Mission) Music Theme [jIGpIBiaZb0].opus'
Deleting original file Grand Theft Auto [GTA] V - Prologue (1st Mission) Music Theme [jIGpIBiaZb0].webm (pass -k to keep)
[download] Downloading item 4 of 230
[youtube] Extracting URL: https://www.youtube.com/watch?v=Xa3OayDkcr4
[youtube] Xa3OayDkcr4: Downloading webpage
^C
ERROR: Interrupted by user
```

**Comment by pukkandan** [comment]
ah, I see what's happening - it's https://github.com/yt-dlp/yt-dlp/issues/4876. Use `--audio-format opus` to workaround.
